examples/design-system-tutorial/src/components/card/card.styles.ts

```diff
@@ -0,0 +1,77 @@
+import {
+    controlCornerRadius,
+    designUnit,
+    fillColor,
+    neutralForegroundRest,
+    typeRampMinus1FontSize,
+    typeRampMinus1LineHeight,
+    typeRampPlus1FontSize,
+    typeRampPlus1LineHeight,
+    elevation,
+} from "@fluentui/web-components";
+import { css } from "@microsoft/fast-element";
+import { DesignToken, display } from "@microsoft/fast-foundation";
+
+/**
+ * Not only can we have system-wide design tokens, but we can
+ * have component-specific tokens as well. This token enables
+ * changing the corner radius of cards independently of other controls.
+ * However, by default it aliases to the controlCornerRadius token.
+ */
+export const cardCornerRadius = DesignToken.create<number>(
```

Comment:
An example like this is important, but I'm concerned this specific one might be confusing. We have `layerCornerRadius` that's applied to cards by default, so I'd like to avoid the suggestion that to have cards different than "controls" that you need to create new token. This is why in our design token examples page I created something obviously new, I think I called it "special".

What about giving the card `padding` instead and creating a `cardPadding` token? That starts to creep into density, but it's a realistic use case that's come up with cards used across different experiences.